signer/src/testing/dummy.rs

```diff
@@ -161,3 +151,25 @@ pub fn signature<R: rand::RngCore + ?Sized>(
 
     p256k1::ecdsa::Signature::new(&multipurpose_bytes, &secret_key).unwrap()
 }
+
+/// Coinbase transaction with random block height
+fn coinbase_tx<R: rand::RngCore + ?Sized>(
+    config: &fake::Faker,
+    rng: &mut R,
+) -> bitcoin::Transaction {
+    // Numbers below 17 are encoded differently which messes with the block height decoding
+    let min_block_height = 17;
+    let block_height_span = 10000;
+    let block_height =
+        config.fake_with_rng::<i64, _>(rng).abs() % block_height_span + min_block_height;
```

Comment:
If we import `rand::Rng` then this can be simplified to
```rust
let block_height: i64 = rng.gen_range(17..10000);
```

Reply:
Thanks for the recommendation. Updated.